tinyformat_test.cpp

```diff
@@ -113,6 +114,20 @@ void speedTest(const std::string& which)
             fmt::print("{:.10f}:{:04}:{:+}:{}:{}:{}:%\n",
                 1.234, 42, 3.13, "str", (void*)1000, 'X');
     }
+    else if(which == "format::prepare")
+    {
+        // format::prepare version.
+        const auto prepared_format =
+            fmt::prepare<double, int, double, const char*, void*, char>("{:.10f}:{:04}:{:+}:{}:{}:{}:%\n");
+        fmt::basic_memory_buffer<char, 1024> buffer;
+        for(long i = 0; i < maxIter; ++i)
+        {
+            auto finished_at =
+                prepared_format.format_to(buffer, 1.234, 42, 3.13, "str", (void*)1000, 'X');
+            *finished_at = '\0';
+            std::cout << buffer.data();
```

Comment:
Have you tried `puts` instead? It might be even faster.

Reply:
Indeed, `std::puts` seems to be a little faster. I'm going to prepare google benchmark tests when I find some time.